integration/php_httpd_test.go

```diff
@@ -78,6 +83,46 @@ func testPhpHttpd(t *testing.T, context spec.G, it spec.S) {
 			Expect(logs).To(ContainLines(ContainSubstring("Apache HTTP Server Buildpack")))
 			Expect(logs).To(ContainLines(ContainSubstring("PHP Web Buildpack")))
 			Expect(logs).To(ContainLines(ContainSubstring("PHP Composer Buildpack")))
+			Expect(logs).NotTo(ContainLines(ContainSubstring("Procfile Buildpack")))
+		})
+
+		context("when there is a Procfile", func() {
+			it.Before(func() {
+				Expect(ioutil.WriteFile(filepath.Join(source, "Procfile"), []byte("web: procmgr /layers/paketo-buildpacks_php-web/php-web/procs.yml"), 0644)).To(Succeed())
```

Comment:
Could you please use a start command that's not the same as the default one, so it can be asserted later?
Maybe `procmgr /layers/paketo-buildpacks_php-web/php-web/procs.yml --mockarg`